domain-xample/svc-xample-j/src/integrationTest/java/gov/va/vro/services/xample/XampleJavaMicroserviceTest.java

```diff
@@ -32,14 +34,22 @@ public class XampleJavaMicroserviceTest {
 
   @Autowired private RabbitAdmin rabbitAdmin;
 
-  @BeforeEach
-  private void setUp() {
-    rabbitAdmin.purgeQueue(queueName, true);
+  private final RetryTemplate retryTemplate = createRetryTemplate();
+
+  private static RetryTemplate createRetryTemplate() {
+    ExponentialBackOffPolicy backOffPolicy = new ExponentialRandomBackOffPolicy();
+    backOffPolicy.setInitialInterval(2000);
+
+    RetryTemplate retryTemplate = RetryTemplate.defaultInstance();
+    retryTemplate.setBackOffPolicy(backOffPolicy);
+    retryTemplate.setRetryPolicy(new MaxAttemptsRetryPolicy(5));
+    return retryTemplate;
   }
 
-  @AfterEach
-  private void tearDown() {
-    rabbitAdmin.purgeQueue(queueName, true);
+  @BeforeEach
+  private void setUp() {
+    rabbitTemplate.setRetryTemplate(retryTemplate);
```

Comment:
This placement of `setRetryTemplate()` is not ideal, for a few reasons: 
1. it should be executed just once, such as in a `@BeforeAll` function. However, the method signature for [`@BeforeAll`](https://junit.org/junit5/docs/5.0.0/api/org/junit/jupiter/api/BeforeAll.html) is static, which is incompatible with the `rabbitTemplate` instance.
2. it should be configured in spring (or whatever it is that sets up `rabbitTemplate`)

... and I'm stumped on how to implement either of these. Very open to pointers. Thanks.